examples/joystick/Joystick.cpp

```diff
@@ -112,18 +112,18 @@ int main()
     sstr.str("");
     sstr << threshold << "  (Change with up/down arrow keys)";
     {
-        auto [it, success]   = texts.try_emplace("Threshold", JoystickObject{{font, "Threshold:"}, {font, sstr.str()}});
-        auto& [label, value] = it->second;
+        const auto [it, success] = texts.emplace("Threshold", JoystickObject{{font, "Threshold:"}, {font, sstr.str()}});
```

Comment:
@ChrisThrasher Why did try_emplace became emplace?

Reply:
I'm sure that was an oversight during merge conflict resolution. I'll fix that in a local branch of mine so I can get it back into `master` some time in the future. Good catch!